.circleci/config.yml

```diff
@@ -58,6 +58,20 @@ jobs:
       - run: cd build && cmake -DNIGHTLY_BUILD=ON -DCMAKE_TOOLCHAIN_FILE=../CMake/mingwcc.cmake ..
       - run: cd build && cmake --build . -j $(nproc)
       - store_artifacts: {path: ./build/devilutionx.exe, destination: devilutionx_x86.exe}
+  switch:
+    docker:
+      - image: devkitpro/devkita64:latest
```

Comment:
Tool chain is outdated so won't work on firmware versions 9, either update at runtime or use a different image.

updating sdk using dkp-pacman should work